### DISCUSSION TITLE
Can I use for http?

### DISCUSSION THREAD

**Discussion by toweroff** [Theldus/wsServer#63] [General]
I do not need a parsing by wsServer, I want to get http text only and answer to client over ws_sendframe or ws_sendframe_txt
it is possible?

**Comment by Theldus** [comment]
Hi @toweroff,
I don't know if I understood your question correctly, but wsServer doesn't require an HTTP server to work. You can always open the .html files locally in the browser and use them.

It is worth mentioning, however, that wsServer does not work as an HTTP server... if that is the functionality you want, you can use a server like Apache, Nginx or another library together with wsServer.
